lib/src/lsm.rs

```diff
@@ -24,8 +24,13 @@ const SELF_CURRENT: &str = "/proc/self/attr/current";
 
 #[context("Querying selinux availability")]
 pub(crate) fn selinux_enabled() -> Result<bool> {
-    let filesystems = std::fs::read_to_string("/proc/filesystems")?;
-    Ok(filesystems.contains("selinuxfs\n"))
+    let path = "/proc/1/root/sys/fs/selinux/enforce";
+    if Path::new(path).exists() {
+        let enabled = std::fs::read_to_string(path)?;
+        return Ok(enabled.eq("1"));
```

Comment:
It's a bit more idiomatic to do `enabled == "1"`

However...I think we actually *should* work even if `enforce == 0`.  So this check can probably just boil down to

```
Path::new("/proc/1/root/sys/fs/selinux/enforce").try_exists().map_err(Into::into)
```
or so right?

Reply:
> However...I think we actually should work even if enforce == 0

To elaborate slightly, this is the current case in Anaconda - it boots with `enforcing=0`.  Our existing install logic should work just fine in this case.

It's really the case of `selinux=0` (fully disabled at runtime) that we need a fallback behavior.

Reply:
ah that makes sense, if the labels exist then bootc should work. For some reason that is not the case right now (i.e. the labels are broken on the image when installing from a host with selinux set to permissive), I guess that is what @bcrochet is working on?

I fixed the first commit with your suggestions. I also added a second commit to prioritize the `--disable-selinux` flag. It seems wrong to silently ignore the flag if the host has selinux enabled. I'm probably missing something though. Flipping those if statements makes `install to-disk --disable-selinuc` work from a host with selinux set to permissive though.